src/views/components/results-raw-output.html

```
@@ -1,110 +1,174 @@
-										class="c-gp-results-raw-output_list_item_status-line"
-										style="background-color: {{_.getGpProbeStatusColor(this.statsPerTarget[activeTargetIdx].avgTiming, testReqParams.type === 'http' ? 1000 : 200) || ''}}">
-									</span>
-									<img width="28"
-										height="20"
-										src="https://cdn.jsdelivr.net/npm/country-flag-icons@1.5.5/3x2/{{this.country}}.svg">
-										{{this.city}}, {{this.country}}, {{this.continent}}
-									<span>&nbsp;-&nbsp;</span>
-									<a class="c-gp-results-raw-output_list_item_header_info_network-link" href="{{@this.getUrlNetworkName('/networks', this.network)}}">
-										{{this.network}} (AS{{this.asn}})
-									</a>
+			<div class="c-gp-results-raw-output_list_item item-id-{{this.clientSideId}}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)